.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,30 @@
+name: Pull Request
+
+on:
+  pull_request:
+    branches:
+      - master
+      
+jobs:
+  cancelPreviousRuns:
+    name: Cancel previous runs
+    if: github.event_name == 'push' || !github.event.pull_request.draft
+    runs-on: macos-latest
```

Comment:
As this job is running in another VM as the iOS build, you can use `linux-latest` here to reduce the number of build minutes (because macOS costs 10x compared to linux).